dex-auth.go

```diff
@@ -112,9 +115,19 @@ func (cluster *Cluster) handleCallback(w http.ResponseWriter, r *http.Request) {
 	buff := new(bytes.Buffer)
 	json.Indent(buff, []byte(claims), "", "  ")
 
+	if cluster.Config.IDP_Ca_Pem != "" {
```

Comment:
Same comments on naming as before, but I can't say I'm concerned about fixing the style deviation as there are quite a few places with it which are well out of scope.

There is also the condition where both `IDP_Ca_Pem` and `IDP_Ca_Pem_File` are provided, which may be erroneous but is supported in main.go